ci-operator/step-registry/gather/ibmcloud-resource/gather-ibmcloud-resource-commands.sh

```diff
@@ -126,8 +126,18 @@ function gather_cos {
 
 # Gather CIS resources
 function gather_cis {
-    command_retry "${IBMCLOUD_CLI}" cis instance-set "Openshift-IPI-CI-CIS"
-    DOMAIN_ID=$(command_retry "${IBMCLOUD_CLI}" cis domains --per-page 50 | awk '/ci-ibmcloud.devcluster.openshift.com/{print $1}')
+    local cisName BASE_DOMAIN
+    cisName=$(command_retry "${IBMCLOUD_CLI}" cis instances | grep openshift | awk '{print $1}')
```

Comment:
We can save the cis name into the cluster profile's vault (maybe call it `ibmcloud-cis`) , once the key is existing under `CLUSTER_PROFILE_DIR`, then read it the `cisName` from the file. So that the cis can easily switch with cluster profile being used.

Reply:
whether need put the cis domain into the vault ?

Reply:
We only need to know cis domain for the step, right? If right, directly save the domain to `ibmcloud-cis-domain`